packages/core/src/App/Containers/Layout/header/traders-hub-header-wallets.tsx

```diff
@@ -49,41 +51,43 @@ const TradersHubHeaderWallets = observer(() => {
             })}
         >
             <div className='traders-hub-header__menu-left'>
-                <MobileWrapper>
-                    <ToggleMenuDrawer {...{ platform_config: filterPlatformsForClients(platform_config) }} />
-                    {header_extension && is_logged_in && <div>{header_extension}</div>}
-                    <div className={'traders-hub-header__logo-wrapper'}>
+                {!isDesktop && (
+                    <React.Fragment>
+                        <ToggleMenuDrawer {...{ platform_config: filterPlatformsForClients(platform_config) }} />
+                        {header_extension && is_logged_in && <div>{header_extension}</div>}
+                        <div className={'traders-hub-header__logo-wrapper'}>
+                            <div className='traders-hub-header-wallets__logo'>
+                                <StaticUrl href='/'>
+                                    <DerivBrandShortLogo />
+                                </StaticUrl>
+                            </div>
+                        </div>
+                    </React.Fragment>
+                )}
+                {isDesktop && (
+                    <React.Fragment>
                         <div className='traders-hub-header-wallets__logo'>
                             <StaticUrl href='/'>
                                 <DerivBrandShortLogo />
                             </StaticUrl>
                         </div>
-                    </div>
-                </MobileWrapper>
-                <DesktopWrapper>
-                    <div className='traders-hub-header-wallets__logo'>
-                        <StaticUrl href='/'>
-                            <DerivBrandShortLogo />
-                        </StaticUrl>
-                    </div>
-                    <div className='traders-hub-header__divider' />
-                    <TradersHubHomeButton />
-                </DesktopWrapper>
+                        <div className='traders-hub-header__divider' />
+                        <TradersHubHomeButton />
+                    </React.Fragment>
+                )}
                 <MenuLinks {...{ is_traders_hub_routes: true }} />
             </div>
-            <DesktopWrapper>
-                <div className='traders-hub-header__menu-right'>
-                    <div className='traders-hub-header__divider' />
-                    <div className='traders-hub-header__menu-right--items'>
-                        <div className='traders-hub-header__menu-right--items--onboarding'>
-                            <TradersHubOnboarding />
-                        </div>
-                        <div className='traders-hub-header__menu-right--items--notifications'>
-                            <ShowNotifications />
-                        </div>
-                        {isTabletOs ? (
-                            accountSettings
-                        ) : (
+            {isDesktop && (
+                <React.Fragment>
```

Comment:
Any reason why isDesktop and !isDesktop is checked separately and not with a ternary here? @aizad-deriv